tensorflow/tools/ci_build/linux/mkl/build-dev-container.sh

```diff
@@ -201,21 +201,20 @@ function test_container()
   debug "ID of the running docker container: ${CONTAINER_ID}"
 
   debug "Performing basic sanity checks on the running container..."
-  TEST_CMD_1=$(${DOCKER_BINARY} exec ${CONTAINER_ID} bash -c "${PYTHON} -c 'from tensorflow.python import _pywrap_util_port; print(_pywrap_util_port.IsMklEnabled())'")
-  # Make TEST_CMD backward compatible with older code
-  TEST_CMD_2=$(${DOCKER_BINARY} exec ${CONTAINER_ID} bash -c "${PYTHON} -c 'from tensorflow.python import pywrap_tensorflow; print(pywrap_tensorflow.IsMklEnabled())'")
-
-  if [ "${TEST_CMD_1}" = "True" -o "${TEST_CMD_2}" = "True" ] ; then
-      echo "PASS: MKL enabled test in ${TEMP_IMAGE_NAME}"
-  else
-      die "FAIL: MKL enabled test in ${TEMP_IMAGE_NAME}"
-  fi
+  {
+    ${DOCKER_BINARY} exec ${CONTAINER_ID} bash -c "${PYTHON} -c 'from tensorflow.python import _pywrap_util_port; print(_pywrap_util_port.IsMklEnabled())'"
+    echo "PASS: MKL enabled test in ${TEMP_IMAGE_NAME}"
+  } || {
+    ${DOCKER_BINARY} exec ${CONTAINER_ID} bash -c "${PYTHON} -c 'from tensorflow.python import pywrap_tensorflow; print(pywrap_tensorflow.IsMklEnabled())'"
+    echo "PASS: Old MKL enabled in ${TEMP_IMAGE_NAME}"
+  } || {
+    die "FAIL: MKL enabled test in ${TEMP_IMAGE_NAME}"
+  }
```

Comment:
I think for here it makes sense to use `TEST_CMD_1` and `TEST_CMD_2`. Would you mind changing it back? Sorry I wasn't being clear. I wanted to get rid of only `HOROVOD_TEST_CMD` because having it didn't help with readability.

Reply:
I changed that code as well because the previous code was actually running TEST_CMD_2 as well and was hence throwing a warning:
"Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'tensorflow.python.pywrap_tensorflow' has no attribute 'IsMklEnabled'"

Reply:
I see. Let's go with this then. Thank you again!

Reply:
Thank you for reviewing and providing feedback. Appreciate it.